examples/rust/raw_mesh/src/main.rs

```diff
@@ -48,10 +48,14 @@ impl From<GltfPrimitive> for Mesh3D {
         if let Some(vertex_colors) = vertex_colors {
             mesh = mesh.with_vertex_colors(vertex_colors);
         }
+        if let Some(vertex_texcoords) = vertex_texcoords {
+            mesh = mesh.with_vertex_texcoords(vertex_texcoords);
+        }
         if albedo_factor.is_some() {
             mesh = mesh.with_mesh_material(rerun::datatypes::Material {
                 albedo_factor: albedo_factor
                     .map(|[r, g, b, a]| ecolor::Rgba::from_rgba_unmultiplied(r, g, b, a).into()),
+                albedo_texture: None, // TODO(andreas): This would require loading the right texture file and converting it to `TensorData`.
```

Comment:
Why not do it for the rust example too?

Reply:
I ran out of time and didn't see a quick & easy path towards that. The gltf loader doesn't load the image itself, so I need an extra path that loads the images from disk etc.